src/components/EncounterNotes/MursaatOverseerNotes.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function MursaatOverseerNotes() {
+  return (
+    <div>
+      <h3>General</h3>
+      <ul>
+        <li>Claim: Blaster</li>
```

Comment:
Could be nice to add the special action key skills? Can do this later though. This is completely fine for now.

Reply:
Wiki doesn't list MO special action key skills in API unfortunately. You'd have to make a custom component to show icons just for that or something.